src/models/FaresQuery.tsx

```diff
@@ -0,0 +1,5 @@
+export interface FaresQuery {
+    originStationCrs: string;
+    destinationStationCrs: string;
+    outboundDateTime: Date;
+}
```

Comment:
we want to be adding a newline at the end of all files, again is just a styling convention, github gives this angry red circle when you forget so it's quick to spot when you're looking at your changes